packages/cozy-stack-client/src/PermissionCollection.js

```diff
@@ -105,33 +105,49 @@ class PermissionCollection extends DocumentCollection {
   }
 
   /**
-   * Create a share link
-   *
-   * @param {{_id, _type}} document - cozy document
-   * @param {object} options - options
-   * @param {string[]} options.verbs - explicit permissions to use
+   * @private
    */
-  async createSharingLink(document, options = {}) {
-    const { verbs } = options
+  async createPermissions(permissions) {
```

Comment:
can we have a jsdoc for this `permissions` ?

Reply:
done